pkg/admission/cert/writer/internal/atomic/atomic_writer.go

```diff
@@ -57,8 +56,8 @@ const (
 // inotify or fanotify to receive events when the content in the volume is
 // updated.
 type AtomicWriter struct {
-	targetDir  string
-	logContext string
+	targetDir string
+	log       logr.Logger
```

Comment:
Changing this file means that it starts to diverge from upstream.
But I guess it's not a problem.

Reply:
if we actually have our own copy, we should treat it as such, IMO.